tests/test.cpp

```diff
@@ -568,6 +568,33 @@ void FuzzTest2() {
 
   RndDef definitions[num_definitions];
 
+  // Since we're generating schema and corresponding data in tandem,
+  // this convenience functor adds strings to both at once.
+  struct AddToSchemaAndInstances_t
```

Comment:
Since RndDef is a locally declared struct, also kept AddToSchemaAndInstances as a locally defined struct. However, made this more compact than the previous iteration.